### DISCUSSION TITLE
[Metrics] Influxdb

### DISCUSSION THREAD

**Discussion by bragarods** [JanusGraph/janusgraph#2932] [Ideas]
Would it be nice to have support to write metrics on InfluxDB as we have for Graphite.

**Comment by lionelfleury** [comment]
You can provide your own reporter.
https://docs.janusgraph.org/operations/monitoring/
Using telegraph you can use some approaches from JMX exporter. "Sidecar" e.g.
Same is valid for Prometheus.
